libs/island-ui/theme/src/lib/theme.ts

```diff
@@ -101,8 +101,6 @@ export const theme = {
       large: `${radius.default.$value ?? 8}px`, // TODO: check with designers to keep as large
       md: `${radius.md.$value ?? 12}px`,
       lg: `${radius.lg.$value ?? 16}px`,
-      xl: '24px', // TODO: should be added to figma?
-      circle: '50%', // TODO: should be added to figma?
       full: `${radius.full.$value ?? 9999}px`,
```

Comment:
You need to be sure that this does not break in `react-native`. You should verify with @thoreyjona